File: Python/vowels.py
# Python3 program to count vowels
# in a string

# Function to check the Vowel
def isVowel(ch):
	return ch.upper() in ['A', 'E', 'I', 'O', 'U']

# Returns count of vowels in str 
def countVowels(str):
	count = 0
	for i in range(len(str)):

		# Check for vowel
		if isVowel(str[i]):
			count += 1
	return count

# Driver Code

# string object 
str = 'abc de'

# Total number of Vowels
print(countVowels(str))

File: CPP/vowels.cpp
// CPP program to print count of
// vowels using pointers

#include <iostream>

using namespace std;

int vowelCount(char *sptr)
{
	// Create a counter
	int count = 0;

	// Iterate the loop until null character encounter
	while ((*sptr) != '\0') {

		// Check whether character pointer finds any vowels
		if (*sptr == 'a' || *sptr == 'e' || *sptr == 'i'
			|| *sptr == 'o' || *sptr == 'u') {

			// If vowel found increment the count
			count++;
		}

		// Increment the pointer to next location
		// of address
		sptr++;
	}

	return count;
}

// Driver Code
int main()
{
	// Initialize the string
	char str[] = "geeksforgeeks";

	// Display the count
	cout << "Vowels in above string: " << vowelCount(str);

	return 0;
}
